src/Geometry.h

```diff
@@ -27,6 +27,19 @@
 class Cmfd;
 
 /**
+ * @enum ringifyType
+ * @brief The method to use when subdividing cells into rings
+ */
+enum ringifyType{
+
+  /** Max ring radius chosen such that it has the same area as the total geometry */
```

Comment:
I suppose I don't understand... but the Geometry I'm referring to here is the current geometry... I'm using the width of the Geometry (not of a pin cell).  As far as I know I'm not querying the bounding box of the universe... I'm using the bounds of the current Geometry object.  But maybe I'm misunderstanding the code...